src/api.js

```diff
@@ -8,4 +8,17 @@ export const fetchAllReviews = () => {
     return reviewsURL.get('/reviews').then((response) => {
         return response.data.allReviews
     })
-}
\ No newline at end of file
+}
+
+export const fetchReviewByID = (review_id) => {
+    return reviewsURL.get(`/reviews/${review_id}`).then((response) => {
+        return response.data.ReviewsByID
+    })
+}
+
+export const fetchCommentsByID = (review_id) => {
+    return reviewsURL.get(`/reviews/${review_id}/comments`).then((response) => {
+        if(response.data.length === 0) return []
```

Comment:
This looks like your BE isn't working as expected, as it should always be returning an array on a key of comments for a successful request. I appreciate this is how you're getting around it on the FE, though. Definitely one to go back and look at when you have time for a refactor.